### PR TITLE
Add pysynphot

### PR DESCRIPTION
**Pull request by mfisherlevine**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mfisherlevine** [comment]
Not sure if it's necessary (given I'm filing the PR), but commenting that I'm willing to be listed as maintainer just in case 🙂

**Comment by mfisherlevine** [comment]
@pllim I emailed help@stsci.edu to see if you (or any of the other maintainers of pysynphot) would like to be added as maintainers here (as well as, or instead of, me) but I don't know how quickly they'll pass that along, so thought I'd tag you here in case that's more effective. Apologies for the lack of introduction!

**Comment by pllim** [comment]
Hello. Yes, I am the current maintainer and you can add me, though I am a little taken by surprise here.

Are you affiliated with STScI?

File: recipes/pysynphot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  imports:
+    - pysynphot
+
+about:
+  home: https://github.com/spacetelescope/pysynphot
+  license:  BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.md
+  summary: 'Synthetic photometry package from Astrolib.'
+  doc_url: https://github.com/spacetelescope/pysynphot/tree/master/doc
```
Comment (pllim) [inline recipes/pysynphot/meta.yaml:43]: I think the doc should point to https://pysynphot.readthedocs.io/en/latest/

**Comment by mfisherlevine** [comment]
> Hello. Yes, I am the current maintainer and you can add me, though I am a little taken by surprise here.
> 
> Are you affiliated with STScI?

Hi, sorry about that, there was a bit more preamble in the email!

I'm with AURA, but not STScI, working on Rubin/LSST. We wanted to package this with the LSST codebase so that we can use it in the spectroscopic reduction pipeline for our Auxiliary Telescope, and in order to package it more easily with our software distribution mechanisms, I was asked to put it on conda-forge.

I can make you a maintainer if you'd _like_ but please don't feel obligated, I'm happy to do the work, just thought I'd offer in case you would prefer ownership. The conda-forge docs warn that adding a GitHub ID as a maintainer means there will be spam, so there's that to consider too.

File: recipes/pysynphot/meta.yaml (hunk 1 above)
Comment (mfisherlevine) [reply recipes/pysynphot/meta.yaml:43]: Thanks for pointing this out. I'll look into how to fix this, as I'm totally new to the conda-forge workflow and this PR has sadly already merged (they're _fast_!)

**Comment by pllim** [comment]
> this PR has sadly already merged

Yeah, not sure how to update this thing without another release. 🤷 The doc link isn't critical to fix.

> I can make you a maintainer if you'd like but please don't feel obligated

I do not mind at all. I think adding me would help because I'll get pinged by the bot and if the recipe needs updating, I would know how to fix it since I do the source packaging.

In the meantime, feel free to ping me as needed.

> We wanted to package this with the LSST

Hmm. I have some comments about this, but we can discuss over at the Help Desk.

**Comment by beckermr** [comment]
Changes to the doc string and adding maintainers can be done in the feedstock. They may not be reflected on anaconda.org in some cases.

**Pull request merged**